Runes.xcodeproj/project.pbxproj

```diff
@@ -944,6 +951,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "com.thoughtbot.$(PRODUCT_NAME:rfc1034identifier)";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SDKROOT = macosx;
+				SWIFT_VERSION = 2.3;
```

Comment:
Should this be added to all of the targets (including the tests)? It looks like it's only being added to the frameworks.